.github/workflows/trigger-backend-enterprise-update.yml

```
@@ -0,0 +1,39 @@
+name: Update vendored version in grafana/backend-enterprise
```

[Nit] Maybe the name can be more explicit about what the workflow does?
```suggestion
name: Update vendored version in grafana/backend-enterprise release branches
```